input/fsh/QuestionnaireSEVendorLite.fsh

```diff
@@ -0,0 +1,84 @@
+Profile: QuestionnaireSEVendorLite
```

Comment:
Shall we go with the new naming convention?

Reply:
Fixing

Reply:
Shall we reflect the profile name in fsh file name too?